spring-data-jdbc/src/test/java/org/springframework/data/jdbc/core/convert/SqlGeneratorUnitTests.java

```diff
@@ -778,8 +777,8 @@ void selectByQueryValidTest() {
 		String generatedSQL = sqlGenerator.selectByQuery(query, parameterSource);
 		assertThat(generatedSQL).isNotNull().contains(":x_name");
 
-		assertThat(parameterSource.getValues()) //
-				.containsOnly(entry("x_name", probe.name));
+		assertThat(parameterSource.getValues()).hasSize(1) //
```

Comment:
I really don't like this change. It does essentially the same assertion but in two pieces. I find the original version clearer and easier to understand.

Reply:
Let me roll back those changes to simplify things.